chapters/Chapter 1 — Untitled.md

```diff
@@ -87,38 +86,46 @@ Leanne took an explosive from her belt, and planted it on the hull, just behind
 
 She stepped back. Entered a code on her wrist. It seemed low tech, but that was the way with most weapons. More advanced technology could only be controlled by Alts.
 
-The explosion ripped the front of the cockpit off her attackers’ still-accelerating ship. Leanne hit the hull just before the debris shot over her.
+The explosion ripped the front of the cockpit off her attackers’ still-accelerating ship. Leanne hit the hull just before the debris shot over her head.
 
-Leanne’s other attacker flew out of the hole. A bullet followed her. Leanne supposed it hadn’t been necessary: the other attacker wasn’t wearing a suit. She’d be out in thirty seconds, and dead in another thirty, unless her Alt could pull off a miracle. But Leanne’s rule was Leanne’s rule.
+Leanne’s other attacker flew out of the hole. A bullet followed them. Leanne supposed it hadn’t been necessary: the other attacker wasn’t wearing a suit. She’d be unconscious in thirty seconds, and dead in another thirty, unless her Alt could pull off a miracle. But Leanne couldn’t take chances. She had to see it.
```

Comment:
don't need the comma between "another thirty" and "unless"

If I change above to mention she had to be certain, then I could leave this at:

> But Leanne had to see it.